auto/src/glew_tail.h

```diff
@@ -8,10 +8,10 @@
 #define GLEW_ERROR_GLX_VERSION_11_ONLY 3  /* Need at least GLX 1.2 */
 
 /* string codes */
-#define GLEW_VERSION 1
-#define GLEW_VERSION_MAJOR 2
-#define GLEW_VERSION_MINOR 3
-#define GLEW_VERSION_MICRO 4
+#define GLEW_VERSION GLEW_VERSION_MAJOR_STRINGGLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MAJOR GLEW_VERSION_MAJOR_STRING
+#define GLEW_VERSION_MINOR GLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MICRO GLEW_VERSION_MICRO_STRING
 
```

Comment:
Maybe you are right. I just haven't thought about such weird case when someone could use library version different from header version. It's really not a way to go. It's the same as pkg-config returning not matching version. Whole point is to check version from header at configure time so you can make some decisions in build scripts. And many open-source libraries seem to be ok defining version info like header constants. 

In the end of the day defining version info in comments can be an option. Honestly I don't like it but it's probably a matter of taste. Maybe we can introduce another 4 defines like a compromise. Anyway you are boss here it's up to you to decide. Just give us that ability to grep version from header.